flink-scala/src/main/scala/org/apache/flink/api/scala/DataSet.scala

```diff
@@ -89,7 +89,7 @@ class DataSet[T: ClassTag](set: JavaDataSet[T]) {
   /**
    * Returns the TypeInformation for the elements of this DataSet.
    */
-  def getType(): TypeInformation[T] = set.getType()
```

Comment:
We need to actually define in the code style whether we call Java functions with or without parenthesis.

Last time we discussed that, we concluded that since Java functions do not declare side-effect-freeness, may be good to always keep the parenthesis. Plus, for getters this always seemed strange to me. Methods without parenthesis are Scala's style of blurring the boundaries between methods and variables. But for something that is explicitly named like a method (`getType()` rather than `type()`) it looks weird to try and treat it like a variable.